mods/ra/installer/remaster.yaml

```diff
@@ -0,0 +1,329 @@
+ra-remaster-steam: C&C Remastered (Steam version, English)
+	Type: Install
+	RegistryPrefixes: HKEY_LOCAL_MACHINE\SOFTWARE\
+	RegistryKey: Microsoft\Windows\CurrentVersion\Uninstall\Steam App 1213210
```

Comment:
I'd prefer if we could expose this as a distinct `SteamAppID` key, with new install logic that knows to look at this registry key on windows or do whatever is needed to find the path into the proton container on Linux.

This feature will also be used by the KKnD mod.